src/v2/Apps/Artist/Routes/WorksForSale/Components/ZeroState.tsx

```diff
@@ -13,14 +13,13 @@ export const ZeroState: React.FC = () => {
   }
 
   return (
-    <Box width="100%" my={1}>
+    <Box width="100%">
       <Sticky>
         {({ stuck }) => {
           return (
-            <Box pt={stuck ? 1 : 0}>
-              <Message>
-                <Text>No works available by the artist at this time</Text>
-                <Text textColor="black60">
+            <Box pt={stuck ? 1 : 0} my={1}>
+              <Message title="No works available by the artist at this time">
+                <Text variant="sm" textColor="black60">
```

Comment:
This `Text` node shouldn't be needed at all. Just simple children here.

Reply:
The reason I went with text here is due to the [figma specs ](https://www.figma.com/file/zG9cduJdOaTQAw886QcOwn/Filters?node-id=3914%3A33534) in order to have `textColor="black60"`

Reply:
Design System > Figma specs. That Figma is just incorrectly specified.

Reply:
https://www.figma.com/file/m6gDpKHEWDbYJyrwsVZDBr/Artsy-3.0.0-Design-System?node-id=1020%3A781

Reply:
In general: if you see yourself doing some one-off modification, bring it up with the design team that they are deviating from the design system.

Reply:
sounds great! Thanks a lot!

Reply:
Also interesting: https://www.figma.com/file/gZNkyqLT8AU3T61tluVJyB/%F0%9F%9A%AB-Artsy-3.1-Design-System?node-id=6726%3A1796

It looks like in 3.1 this does get updated to look like what you're aiming for. But just using the normal `Message` component as it's intended will ensure that when this gets updated it the changes will be correct everywhere.